examples/2DVisualization.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>2D Visualization example</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./assets/css/examples.css" />
+  </head>
+  <body>
+    <script src="../packages/browser/dist/RUN_MODE/bundle.js"></script>
+
+    <script type="text/javascript">
+      udvizBrowser.loadMultipleJSON([
+        './assets/config/extent_lyon.json',
+        './assets/config/frame3D_planars.json',
+        './assets/config/layer/base_maps.json',
+        './assets/config/layer/elevation.json',
+        './assets/config/layer/3DTiles.json',
+      ]).then((configs) => {
+        // http://proj4js.org/
+        // define a projection as a string and reference it that way
+        // the definition of the projection should be in config TODO_ISSUE
+        udvizBrowser.proj4.default.defs(
+          configs['extent_lyon'].crs,
+          '+proj=lcc +lat_1=45.25 +lat_2=46.75' +
+            ' +lat_0=46 +lon_0=3 +x_0=1700000 +y_0=5200000 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs'
+        );
+
+        const extent = new udvizBrowser.itowns.Extent(
+          configs['extent_lyon'].crs,
+          parseInt(configs['extent_lyon'].west),
+          parseInt(configs['extent_lyon'].east),
+          parseInt(configs['extent_lyon'].south),
+          parseInt(configs['extent_lyon'].north)
+        );
+
+        const frame3DPlanar = new udvizBrowser.Frame3DPlanar(
+          extent,
+          configs['frame3D_planars'][2]
+        );
+
+        // ADD BASE MAP
+        udvizBrowser.addBaseMapLayer(
+          configs['base_maps'][0],
+          frame3DPlanar.itownsView,
+          extent
+        );
+
+        // ADD ELEVATION LAYER
+        udvizBrowser.addElevationLayer(
+          configs['elevation'],
+          frame3DPlanar.itownsView,
+          extent
+        );
+
+        // ADD LAYERS
+        udvizBrowser.add3DTilesLayers(
+          configs['3DTiles'],
+          frame3DPlanar.itownsView
+        );
+
+        const button = document.createElement('button');
+        button.innerHTML = 'Change visualization mode to 3D';
+        button.style.zIndex = 2;
+        button.style.position = 'relative';
+        button.style.cursor = 'pointer';
+        frame3DPlanar.appendToUI(button);
+
+        const planarLayer = frame3DPlanar.itownsView.tileLayer;
+        const elevationLayer = frame3DPlanar.itownsView
+          .getLayers()
+          .filter((el) => el.isElevationLayer)[0];
+
+        elevationLayer.scale = 0;
+
+        const change3DTilesHeight = () => {
```

Comment:
the name of this function should be more an update that a change since it can be applied to a 3DTiles without changing anything